tests/lightning_qubit/test_measurements_class.py

```diff
@@ -465,19 +476,40 @@ def test_single_return_value(self, measurement, observable, lightning_sv, tol):
             if isinstance(observable, list)
             else [measurement(op=observable)]
         )
-        tape = qml.tape.QuantumScript(ops, measurements)
+        tape = qml.tape.QuantumScript(ops, measurements, shots=shots)
 
-        expected = self.calculate_reference(tape, lightning_sv)
         statevector = lightning_sv(n_qubits)
         statevector = statevector.get_final_state(tape)
         m = LightningMeasurements(statevector)
-        result = m.measure_final_state(tape)
+
+        skip_list = (
+            qml.ops.Sum,
+            # qml.Hamiltonian,
```

Comment:
Do we want to remove ``# qml.Hamiltonian,``?

Reply:
yes, I will remove :+1: